docs/triggerbindings.md

```diff
@@ -29,12 +29,12 @@ spec:
 
 ## Event Variable Interpolation
 
-In order to parse generic events as efficiently as possible, [GJSON](https://github.com/tidwall/gjson) 
+In order to parse generic events as efficiently as possible, [GJSON](https://github.com/tidwall/gjson)
 is used internally. As a result, the binding [path syntax](https://github.com/tidwall/gjson#path-syntax)
 differs slightly from standard JSON. As of now, the following patterns are
 supported within `TriggerBinding` parameter value interpolation:
-- `$(body(.[0-9A-Za-z_-]+)*)`
-- `$(header(.[0-9A-Za-z_-]+)?)`
+- `$(body([.|][[:alnum:]*?_-]+|[.|]#|[.|]#\([[:alnum:]=<>%!*?"_-]+\)#??)*)`
+- `$(header([.|][:alnum:]+)?)`
```

Comment:
This could be a naive question, but what does `[.|]` mean?

Also, do you think it might be more clear to include the actual regex
```
regexp.MustCompile(`\$\(body(\.[[:alnum:]/_\-\.\\]+|\.#\([[:alnum:]=<>%!"\*_-]+\)#??)*\)`)
```
in the docs instead of using a pseudo regex?

Reply:
Nice catch, I removed [.|] from the actual regex but forgot to remove it from the docs.
GJSON allows you to traverse dicts using `.` or `|`, but I removed `|` to minimize the surface of GJSON syntax exposed.
I will fix the docs with the actual regex.